_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+
+The output tells us that we are looking at an `xarray.DataArray`, with `1` band, `1830` rows, and `1830` columns. We can also see the number of pixel values in the `DataArray`, and the type of those pixel values, which is floating point, or (`float64`). The `DataArray` also stores different values for the coordinates of the `DataArray`. When using `rioxarray`, the term coordinates refers to spatial coordinates like `x` and `y` but also the `band` coordinate. Each of these sequences of values has its own data type, like `float64` for the spatial coordinates and `int64` for the `band` coordinate.
+
+This `DataArray` object also has a couple of attributes that are accessed like `.rio.crs`, `.rio.nodata`, and `.rio.bounds()`, which contain the metadata for the file we opened. Note that many of the metadata are accessed as attributes without `()`, but `bounds()` is a function and needs parentheses. 
+
+~~~
+print(raster_ams_b9.rio.crs)
+print(raster_ams_b9.rio.nodata)
+print(raster_ams_b9.rio.bounds())
+print(raster_ams_b9.rio.width)
+print(raster_ams_b9.rio.height)
+~~~
+{: .language-python}
+~~~
+EPSG:32631
+0
+(600000.0, 5790240.0, 709800.0, 5900040.0)
+1830
+1830
+~~~
+{: .output}
+
+The Coordinate Reference System, or `raster_ams_b9.rio.crs`, is reported as the string `EPSG:32631`. The `nodata` value is encoded as 0 and the bounding box corners of our raster are represented by the output of `.bounds()` as a `tuple` (like a list but you can't edit it). The height and width match what we saw when we printed the `DataArray`, but by using `.rio.width` and `.rio.height` we can access these values if we need them in calculations.
+
+We will be exploring this data throughout this episode. By the end of this episode, you will be able to understand and explain the metadata output.
+
+> ## Data Tip - Object names
+> To improve code readability, file and object names should be used that make it clear what is in the file. The data for this episode covers Amsterdam, and is from Band 9, so we'll use a naming convention of `raster_ams_b9`.
+{: .callout}
+
+## Visualize a Raster
+
+After viewing the attributes of our raster, we can examine the raw values of the array with `.values`:
+
+~~~
+raster_ams_b9.values
+~~~
+{: .language-python}
+~~~
+array([[[    0,     0,     0, ...,  8888,  9075,  8139],
+        [    0,     0,     0, ..., 10444, 10358,  8669],
+        [    0,     0,     0, ..., 10346, 10659,  9168],
+        ...,
+        [    0,     0,     0, ...,  4295,  4289,  4320],
+        [    0,     0,     0, ...,  4291,  4269,  4179],
+        [    0,     0,     0, ...,  3944,  3503,  3862]]], dtype=uint16)
+~~~
+{: .output}
+
+This can give us a quick view of the values of our array, but only at the corners. Since our raster is loaded in python as a `DataArray` type, we can plot this in one line similar to a pandas `DataFrame` with `DataArray.plot()`.
+
+~~~
+raster_ams_b9.plot()
+~~~
```

Comment:
The band as titled in the image is different from the band from sentinel 2 data (this is also what is referred to as band in Episodes 1-4). This could be confusing - a note explaining the two different bands could be useful.